Describe the current development style in README. (#17)
rust-lang-ja/ac-library-rs: README.md

# ac-library-rs

> ac-library-rs a rust port of AtCoder Library(ACL).

Reviewer: ```suggestion
ac-library-rs is a rust port of AtCoder Library (ACL).
```

> See below for ACLs.

Reviewer: ```suggestion
See below for ACL.
```

- [AtCoder Library (ACL) - AtCoder](https://atcoder.jp/posts/517)
- [AtCoder Library - Codeforces](https://codeforces.com/blog/entry/82400)

## Progress

> Out first milestone is 2020/09/20, ACL contest planed.

Reviewer: Probably:
```suggestion
Our first milestone is 2020/09/20, when the first ACL contest is planned.
```

We aim to implement the same toolset as original ACL(C++) on the day.

We are trying to implement with the same files by referring to the original ACL's hpp.

> After that we try to improve develop flow, tests, or utility.

Reviewer: nit:
```suggestion
After that we try to improve our development flow, tests, or utilities.
```

## How to contribute

1. Create an issue or a draft PR before you start working.
2. Make a PR and pass tests.

> ## More infomation

Reviewer: nit:
```suggestion
## More information
```
